shared/src/commonTest/kotlin/org/hyperskill/StudyPlanWidgetTest.kt

```diff
@@ -254,6 +319,22 @@ class StudyPlanWidgetTest {
             formattedTimeToComplete = null
         )
 
+    private fun studyPlanStub(
```

Comment:
In the future, if we will write more unit tests we can make factories for domain models or something like test utils.

Reply:
I searched for a multiplatform mock library but didn't find anything.
There is only a JVM library. Maybe in the future, a multiplatform one appear.